core/tauri-runtime-wry/src/lib.rs

```diff
@@ -3400,13 +3398,10 @@ fn create_window<T: UserEvent, F: Fn(RawWindow) + Send + 'static>(
         .window
         .inner_size_constraints
         .clamp(desired_size, scale_factor)
-        .to_logical::<i32>(scale_factor);
-      let screen_size = monitor.size().to_logical::<i32>(scale_factor);
-      let monitor_pos = monitor.position().to_logical::<i32>(scale_factor);
-      let x = (screen_size.width - window_size.width) / 2 + monitor_pos.x;
-      let y = (screen_size.height - window_size.height) / 2 + monitor_pos.y;
-
-      window_builder = window_builder.position(x as f64, y as f64);
+        .to_physical::<u32>(scale_factor);
+      let position = calculate_window_center_position(window_size, monitor);
+      let logical_position = position.to_logical::<f64>(scale_factor);
```

Comment:
I'll take a look

Reply:
With `AdjustWindowRect`, I think we can even calculate the menu bar size but menu is behind private `platform_specific` in `WindowBuilder` from `tao`, so maybe not

Reply:
If you want to use `AdjustWindowRect`, then treat menu as always `false` when creating the window because the menu is added later after the window is created using `muda` crate.